scripts/asm-installer/install_asm

```diff
@@ -733,6 +748,10 @@ validate_args() {
       fatal "Citadel is not supported with managed control plane."
     fi
 
+    if [[ "${CA}" == "private_ca" ]]; then
+      fatal "Google Certificate Authority Service integration is not supported with managed control plane."
+    fi
+
     if [[ "${CUSTOM_CA}" -eq 1 ]]; then
       fatal "Specifying a custom CA with managed control plane is not supported."
     fi
```

Comment:
Is private CA not a custom CA?

Reply:
It isn't. Custom CA relies on Citadel's CA logic to sign certificates by using custom key/cert's uploaded by the user. Private CA is a completely different gRPC CA service that signs certificates.